docs/policy/static-pools.md

```diff
@@ -35,50 +35,142 @@ CMK compatibility features:
+order to change the active policy cri-resmgr needs to be restarted.
+
+### Pools Configuration
+
+There are three possible sources of the pools configuration, in decreasing
+priority order:
 
-The STP policy tries to use configuration received from the `cri-resmgr-agent`
-which is, in turn, read from a Kubernetes ConfigMap.
+1. CRI-RM global config
+1. stand-alone static-pools config file
+1. CMK directory tree
 
-As a fallback, the STP policy also supports configuration directory format of
-the original CMK. It tries to read the configuration from a location specified
-with `-static-pools-conf-dir` (`/etc/cmk` by default).
-Alternatively, you can provide a configuration file (YAML) by using the
-`-static-pools-conf-file` flag.
-See the
-[example config](/sample-configs/static-pools-policy.conf.example)
-for a starting point.  However, if `cri-resmgr` at a later time receives a
-valid configuration from the `cri-resmr-agent` this will override the fallback
-configuration read from the directory or file.
+The configuration is fully evaluated whenever a re-configuration event is
+received (e.g. from the node agent). Thus, a valid pools config appearing in
+the CRI-RM global config will take precedence over a directory tree based
+config that was previously active. Similarly, removing pools config from the
+CRI-RM global config will make a local config (file or directory tree)
+effective.
 
-**NOTE:** cri-resmgr does not have any utility for automatically generating a
+**NOTE:** cri-resmgr does not have any utility for generating a pool
 configuration. Thus, you need to either manually write one by yourself, or, run
 the `cmk init` command (of the original CMK) in order to create a legacy
 configuration directory structure.
 
-### Install cri-resmgr
+#### Global Config
 
-Deploy cri-resmgr on each node as you would for any other policy.
+Configuration from the global CRI-RM config takes the highest preference, if
+specified (under `policy.static-pools.pools`). A referential example:
 
-### Deploy Admission Webhook
+```yaml
+policy:
+  static-pools:
+    pools:
+      exclusive:
+        exclusive: true
+        cpuLists:
+        ...
+      shared:
+        cpuLists:
+        ...
+      infra:
+        cpuLists:
+        ...
+
+```
 
-You need to run and enable the cri-resmgr mutating admission webhook which is
-making resource request annotations. This is required so that the STP policy is
-able to inspect the extended resources (in this case, exclusive CPU cores)
-requested by containers. See the [webhook](../webhook.md) for instructions.
+#### Stand-alone YAML File
 
-## Running cri-resmgr
+Path to a stand-alone configuration file can be specified by the
+`policy.static-pools.ConfFilePath` option (empty by default) in the CRI-RM
+global config:
 
-An example of running cri-resmgr with the STP policy enabled, creating legacy
-CMK node label and taint:
+```yaml
+policy:
+  static-pools:
+    ConfFilePath: "/path/to/conf.yaml"
```

Comment:
`ConfFilePath` is implemented in commit 1451d5677092ce47747184dd757b65f8c2c2b372 (PR #489: static-pools: slight refactoring and renaming), but that was not backported.

Any ideas which route to take? Backport more or edit docs?

Reply:
If it looks like a documentation bug (documenting something we haven't merged to this branch), then I'd say we should file one more PR fixing the docs.

Reply:
> `ConfFilePath` is implemented in commit [1451d56](https://github.com/intel/cri-resource-manager/commit/1451d5677092ce47747184dd757b65f8c2c2b372) (PR #489: static-pools: slight refactoring and renaming), but that was not backported.

Woops, good catch! See #533